lib/between_meals/knife.rb

```diff
@@ -80,6 +93,20 @@ def cookbook_upload(cookbooks)
       end
     end
 
+    def berks_cookbook_upload(cookbooks)
+      # cookbooks: array
+      # cookbook_paths: array
+      if cookbooks.any?
+        @cookbook_dirs.each do |path|
+          cookbooks.each do |cb|
+            @logger.warn("Running berkshelf on cookbook: #{cb}")
```

Comment:
`exec!` won't work here, because the cookbook may not be in that directory - you will fail for every upload you try to do (assuming you have more than one cookbook directory like we do)